src/css/styles.css

```diff
@@ -1,5 +1,253 @@
-body, html {
-  background-image: linear-gradient(to top, #ff9a9e 0%, #fecfef 99%, #fecfef 100%);
+html {
   background-repeat: no-repeat;
+  background-color: #ebebeb;
+}
+
+html, 
+body {
+  margin: 0;
+  padding: 0;
   height: 100%;
-}
\ No newline at end of file
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+@media only screen and (min-width:768px) {
+  main {
+    flex-direction: row;
+    height: 100vh;
+  }
+}
+
+main section.user-dashboard {
+  display: flex;
+  flex-direction: column;
+  min-height: auto;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 10px; 
+}
+/* ************ */
+@media only screen and (min-width:600px) {
+  header {
+    justify-content: space-between; 
+  }
+}
+/* ************ */
+header .logo {
+  display: flex;
+  align-items: left;
+  flex-grow: 0;
+  margin: 0;
+  padding: 5px;
+  border: 1px solid rgba(49, 49, 49, 0.979);
+  border-radius: 10px;
+  font-size: 14px;
+  color: #949483f0;
+}
+
+@media only screen and (min-width:600px) {
+  header .logo {
+  align-items: center;
+  }
+}
+
+/* *************************** */
+ header .user-info #helloUser {
+  text-align: center;
+  margin-top: 45px;
+  padding-left: 8px;
+  font-size: 18px;
+  color: #674292e8;
+}
+
+@media only screen and (min-width: 600px) {
+  header .user-info #helloUser {
+    padding-left: 0px;
+    font-size: 40px;
+  }
+}
+/* ********************* */
+header h3 {
+  text-align: center;
+  padding-left: 8px;
+  font-size: 8px ;
+  font-weight: normal;
+  color:#4c4d4f;
+}
+
+@media only screen and (min-width: 600px) {
+  header h3 {
+    padding:0;  
+    font-size: 12px;
+  }
+}
+
+.user-info {
+  flex-grow: 1;
+  margin-left: 20px;
+  border: 1px solid black;
+  border-radius: 10px; 
+  position: relative;
+  z-index: 2;
+}
+
+.user-info > span {
+  right: 20px;
+  top: 50%;
+  position: absolute;
+  transform: translateY(-50%);
+}
+
+.user-info >span #calendar-input {
```

Comment:
This selector (and some others below) needs a space between  the ">" and "span" I believe.

Reply:
Just finished those spaces :)